Makefile

```diff
@@ -188,6 +188,15 @@ test/addons/.buildstamp: config.gypi \
 # TODO(bnoordhuis) Force rebuild after gyp update.
 build-addons: $(NODE_EXE) test/addons/.buildstamp
 
+ifeq ($(OSTYPE),$(filter $(OSTYPE),darwin aix))
+  XARGS = xargs
+else
+  XARGS = xargs -r
+endif
+clear-stalled:
+	ps awwx | grep Release/node | grep -v grep | cat
+	ps awwx | grep Release/node | grep -v grep | awk '{print $$1}' | $(XARGS) kill
```

Comment:
@mhdawson If it did, the CI run for this would have failed hard!

On AIX and OS X (which have BSD-based `xargs`), empty stdin is a no-op so `kill` is not called. On all others in CI (which have GNU-based `xargs`), we add `-r` (a few lines above) which causes the same behavior.